.gitignore

```diff
@@ -1 +1,3 @@
 node_modules/
+.vscode
```

Comment:
This can go in the global gitignore, not local. 

`.env` can go there too (that's where it is in my setup) but that one is not a blocker.